stories/Footer.story.jsx

```diff
@@ -1,55 +1,172 @@
-import React, { Fragment } from "react";
+import React from "react";
 import styled from "styled-components";
 import { storiesOf } from "@storybook/react";
 import { withInfo } from "@storybook/addon-info";
-import { Footer } from "../src/index";
+import { Footer, Row, Col } from "../src/index";
 
-const LayoutStyles = styled.div`
-  width: 80%;
-  margin: 0 auto;
-
-  .layout-body {
-    padding: 100px;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-  }
+const Container = styled.a`
+  margin-bottom: 35px;
+  margin-top: 35px;
 `;
 
-const footerChildren = (
-  <Fragment>
-    <a
-      className="link"
-      href="https://www.vivy.com/impressum"
-      target="_"
-      rel="noopener noreferrer"
-    >
-      Impressum
-    </a>
-    <a
-      className="link"
-      href="https://www.vivy.com/nutzungsbedingungen"
-      target="_"
-      rel="noopener noreferrer"
-    >
-      AGB
-    </a>
-    <a
-      className="link"
-      href="https://www.vivy.com/datenschutz"
-      target="_"
-      rel="noopener noreferrer"
-    >
-      Datenschutz
-    </a>
-  </Fragment>
-);
+const Anchor = styled.a`
+  padding: 0 16px;
+  border-right: 1px solid #cdcdcd;
+`;
+const AnchorLast = styled.a`
+  padding-left: 16px;
+`;
 
-storiesOf("Footer", module).add(
-  "default",
-  withInfo("The raw footer component")(() => (
-    <LayoutStyles>
-      <Footer>{footerChildren}</Footer>
-    </LayoutStyles>
-  ))
-);
+storiesOf("Footer", module)
+  .add(
+    "default",
```

Comment:
let's not call this default, I mean, what does it even mean to be default? Is this the default that's always used (with the links inside), does it mean something else, will this always be imported, dunno. Maybe just Example default footer, Example default styles, etc.

Reply:
Example /w default styles - dunno this is our documentation after all - we shouldn't worry about length but more about description.

Reply:
indeed. I renamed it to default styles